earth_enterprise/src/fusion/gepublish/geserveradmin.cpp

```diff
@@ -350,6 +352,7 @@ int main(int argc, char* argv[]) {
     options.flagOpt("setecdefault", ec_default_db);
     options.flagOpt("enable_poisearch", enable_poisearch);
     options.flagOpt("enable_enhancedsearch", enable_enhancedsearch);
+    options.flagOpt("serve_wms", serve_wms);
```

Comment:
We probably ought to have a check at some point to verify that serve_wms is not set unless publishdb is set.

Reply:
This option is only used in PublishDatabase call. See line 654.

Did you mean to check this and report to the user that an invalid option combination was set?

Reply:
Yes, that's the idea. I like having a check at the beginning that verifies that invalid option combinations are not used.

Reply:
Added code at line 408.